Sources/MockoloFramework/Operations/UniqueModelGenerator.swift

```diff
@@ -70,3 +72,33 @@ private func generateUniqueModels(key: String,
     
     return ResolvedEntityContainer(entity: resolvedEntity, paths: paths, imports: pathToContentList)
 }
+
+private func combinePostLookup(models: [Model]) {
```

Comment:
This should be called from ClassModel.applyClassTemplate in the beginning, to ensure all models are unique including the 'processed' models (that can be passed down from mock input files). See how typealias collisions are handled there for example.

Reply:
Moved it + renamed